tutorial/local-setup.md

```diff
@@ -4,27 +4,33 @@ This appendix describes how to check out, build and view the PHP documentation l
 Viewing results as a php.net mirror isn't a simple process, but it can be done.
 The following is one route, and it assumes:
 
-- PHP 5.4+ is available
+- PHP 7.2+ is available
 - Git version control system is available
+- DOM, libXML2, XMLReader, and SQLite3 are available
 - A basic level of shell/terminal usage, or know that shell commands follow a `$` below
 
 If you're interested in simply setting up a local PHP mirror (and NOT build the documentation) then
 please follow the php.net [mirroring guidelines](http://php.net/mirroring) and ignore this document.
 
 ## Checkout the php documentation from Git
-**Assumptions**: This assumes using `/tmp` as the root directory, and checkout of the English language.
-Adjust accordingly, including paths, especially if you are on Windows.
+**Assumptions**: A working directory `phpdoc` is created and will contain the necessary cloned repositories. This tutorial will reference the directories that repositories are cloned into. Adjust as needed, including paths, especially if you are on Windows. This tutorial will clone at least three repositories:
+ * doc-en (into the `en` directory)
+ * doc-base 
+ * phd
 
+If setting up a local php.net mirror is desired, the `web-php` repository must also be cloned. This is explained in the "Set up a local php.net mirror" section.
+
+Note that `doc-en` is cloned into the `en` directory below.
 ```
-$ mkdir /tmp/git
-$ cd /tmp/git
-$ git clone https://github.com/php/doc-en.git
-$ cd /tmp/git/doc-en
+$ mkdir phpdoc
+$ cd phpdoc
+$ git clone https://github.com/php/doc-en.git en
+$ git clone https://github.com/php/doc-base.git
 ```
 
 ## Validate the PHP Documentation XML
 ```
-$ cd /tmp/git/doc-en
+$ cd phpdoc
```

Comment:
Nit: I'm on the fence as to whether each of these snippets should start with `cd phpdoc`, especially if the preceding instructions would land the user in that directory.

Reply:
I went ahead and removed them. I agree, it feels a bit silly to have them, especially if the reader goes through the tutorial in one sitting.